pcsx2/GS/GSLzma.cpp

```diff
@@ -126,107 +126,95 @@ bool GSDumpFile::ReadFile(Error* error)
 		return false;
 	}
 
+	// Get remaining file size for packet data preallocation
+    const s64 current_pos = FileSystem::FTell64(m_fp.get());
```

Comment:
Tabs should be used for indentation.

Reply:
Was working on my macbook and looks like the indents got messed up. Will edit it on my PC !

Reply:
Ok I think it is indented properly now @lightningterror